### PR TITLE
 discovery backplane prow updates

### PR DESCRIPTION
**Pull request by cameronmwall**
Signed-off-by: Cameron Wall <cwall@redhat.com>

### CODE REVIEW

File: ci-operator/config/stolostron/discovery/stolostron-discovery-backplane-2.0.yaml [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+- dockerfile_path: Dockerfile.testserver.prow
+  from: base
+  inputs:
+    stolostron_builder_go1.17-linux:
+      as:
+      - registry.ci.openshift.org/stolostron/builder:go1.17-linux
+  to: discovery-testserver
+promotion:
+  disabled: true
+  name: "2.5"
```
Comment (JakobGray) [inline ci-operator/config/stolostron/discovery/stolostron-discovery-backplane-2.0.yaml:40]: Should this be "2.0"?

File: ci-operator/config/stolostron/discovery/stolostron-discovery-backplane-2.0.yaml [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+  steps:
+    dependencies:
+      SOURCE_IMAGE_REF: discovery-testserver
+    env:
+      IMAGE_REPO: discovery-testserver
+      REGISTRY_ORG: stolostron
+    test:
+    - as: publish
+      commands: |-
+        export OSCI_COMPONENT_NAME="discovery-testiserver"
```
Comment (JakobGray) [inline ci-operator/config/stolostron/discovery/stolostron-discovery-backplane-2.0.yaml:139]: typo here

File: ci-operator/config/stolostron/discovery/stolostron-discovery-backplane-2.0.yaml (hunk 1 above)
Comment (cameronmwall) [reply ci-operator/config/stolostron/discovery/stolostron-discovery-backplane-2.0.yaml:40]: Based on the example of the "work" directory. I think when we are still promoting to both release-2.5 and backplane-2.0, it should stay as "2.5". I think that will change once we remove it after the transition

File: ci-operator/config/stolostron/discovery/stolostron-discovery-backplane-2.0.yaml (hunk 2 above)
Comment (cameronmwall) [reply ci-operator/config/stolostron/discovery/stolostron-discovery-backplane-2.0.yaml]: fixed

File: ci-operator/config/stolostron/discovery/stolostron-discovery-backplane-2.0.yaml (hunk 1 above)
Comment (ray-harris) [reply ci-operator/config/stolostron/discovery/stolostron-discovery-backplane-2.0.yaml:40]: The promotion name/namespace is just to be able to keep different releases separate. It should stay `2.5` for now. I'll talk with David about what needs to happen after the MCE components are removed from ACM.

**Review by JakobGray — APPROVED** [review]

**Pull request merged**